dlc/src/main/scala/org/bitcoins/dlc/testgen/DLCTestVectorGen.scala

```diff
@@ -0,0 +1,89 @@
+package org.bitcoins.dlc.testgen
+
+import java.io.{File, PrintWriter}
+
+import play.api.libs.json._
+
+import scala.concurrent.{ExecutionContext, Future}
+import scala.io.Source
+
+object DLCTestVectorGen {
+  //import DLCTLVGen._
+  import DLCTxGen._
+
+  implicit private val ec: ExecutionContext = ExecutionContext.global
```

Comment:
Instead of using this why not just make `regenerateTestFile()` instead be `regenerateTestFile()(implict ec: ExecutionContext)`

Reply:
I want this the be the object actually used for generating test vectors in "Main" scope (which is what I did to make the json file, in the sbt console)

Reply:
tbh in the future this will likely have a Main branch out of it

Reply:
Oh okay, makes sense